abodepy/helpers/errors.py

```diff
@@ -40,8 +40,8 @@
 INVALID_AUTOMATION_EDIT_RESPONSE = (
     16, "Automation edit response did not match expected values.")
 
-# TRIGGER_NON_QUICKACTION = (
-#     17, "Can not trigger an automation that is not a manual quick-action.")
+SET_PRIVACY_MODE = (
+    17, "Device privacy mode value does not match request value.")
```

Comment:
In the real world you'd probably want to leave the old error commented out with a DEPRECATED note and not re-use the error code number... but it probably doesn't matter very much so I'm OK with this.

Reply:
Wait, are you saying we're living in a simulation right now? :) I kid.

I'll change this. I'd rather adhere to whatever the standard practices are.